nada_dsl/source_ref.py

```diff
@@ -29,7 +34,7 @@ def back_frame(cls) -> "SourceRef":
         """Get the source reference of the calling frame."""
         backend_frame = inspect.currentframe().f_back.f_back
         lineno = backend_frame.f_lineno
-        (offset, length, snippet) = SourceRef.try_get_line_info(backend_frame, lineno)
+        (offset, length, _) = SourceRef.try_get_line_info(backend_frame, lineno)
```

Comment:
maybe we can remove snippet from being returned if we are not using it anywhere

Reply:
done